index.html

```diff
@@ -61,11 +66,12 @@ <h1 class="logo">My Logo</h1>
        <section class="parting">
```

Comment:
Kindly add some space to the left and right of your cards.

![image](https://user-images.githubusercontent.com/34779928/194649046-abeb6103-e590-4370-894a-d3bd6e4a2393.png)